src/span-data.js

```diff
@@ -94,6 +94,10 @@ function SpanData(agent, trace, name, parentSpanId, isRoot, skipFrames) {
  * @returns {SpanData} The new child trace span data.
  */
 SpanData.prototype.createChildSpanData = function(name, skipFrames) {
+  if (this.span.isClosed()) {
```

Comment:
We observed this happening during normal execution for applications that finalize database updates after responding to user requests. I think having this log message available will be good but maybe it makes more sense at `INFO`? `WARN` would probably also be fine but maybe we should change the message to indicate that it may not be a problem?